tdrs-backend/tdpservice/parsers/models.py

```diff
@@ -106,10 +109,10 @@ def get_status(self):
 
         if errors is None:
             return DataFileSummary.Status.PENDING
+        elif precheck_errors.count() > 0 or self.total_number_of_records_created == 0:
+            return DataFileSummary.Status.REJECTED
```

Comment:
this is a minor thing but in the case of a file with valid header/footer and no records, the submission history table reads "Rejected" but generates no error file (second row in this example, as opposed to the first row which contains `1` "Records unable to process" and an error file describing the row precheck error)
<img width="1055" alt="image" src="https://github.com/raft-tech/TANF-app/assets/111305129/cf2a4366-3787-4e82-943b-32f922196dff">
again, minor edgecase. but i wonder if it can be handled by generating a file precheck error within `parse` somewhere

Reply:
Nice catch! I think it makes a lot of sense to generate some type of error to indicate that no records were parsed. I will add that and add a test for this case you found!

Reply:
@jtimpe Have a look a what I did to resolve this case and let me know what you think!